### ISSUE TITLE
Permission Denied Issue

### ISSUE THREAD

**Issue by srinivas1977** [squizlabs/HTML_CodeSniffer#205]
I am trying to do in my local to check for WCAG for my application and i got below errors
npm install grunt-wcag-accessibility 

npm ERR! Error while executing:
npm ERR! /usr/local/bin/git ls-remote -h -t ssh://git@github.com/squizlabs/HTML_CodeSniffer.git
npm ERR! 
npm ERR! fatal: unable to access '/var/root/.gitcinclude': Permission denied
npm ERR! 
npm ERR! exited with error code: 128
Below is the Complete log: 
30 silly fetchPackageMetaData error for HTML_CodeSniffer@github:squizlabs/HTML_CodeSniffer#2.0.7 Error while executing:
30 silly fetchPackageMetaData /usr/local/bin/git ls-remote -h -t ssh://git@github.com/squizlabs/HTML_CodeSniffer.git
30 silly fetchPackageMetaData
30 silly fetchPackageMetaData fatal: unable to access '/var/root/.gitcinclude': Permission denied
30 silly fetchPackageMetaData
30 silly fetchPackageMetaData exited with error code: 128
31 http fetch GET 304 https://registry.npmjs.org/axios 58ms (from cache)
32 http fetch GET 304 https://registry.npmjs.org/bluebird 59ms (from cache)
33 http fetch GET 304 https://registry.npmjs.org/chalk 61ms (from cache)
34 silly pacote range manifest for axios@^0.9.1 fetched in 63ms
35 silly resolveWithNewModule axios@0.9.1 checking installable status
36 http fetch GET 304 https://registry.npmjs.org/commander 62ms (from cache)
37 silly pacote range manifest for bluebird@^3.3.1 fetched in 63ms
38 silly resolveWithNewModule bluebird@3.5.1 checking installable status
39 silly pacote range manifest for chalk@^1.1.1 fetched in 64ms
40 silly resolveWithNewModule chalk@1.1.3 checking installable status
41 silly pacote range manifest for commander@^2.6.0 fetched in 63ms
42 silly resolveWithNewModule commander@2.12.2 checking installable status
43 http fetch GET 304 https://registry.npmjs.org/validator 129ms (from cache)
44 http fetch GET 304 https://registry.npmjs.org/underscore 132ms (from cache)
45 http fetch GET 304 https://registry.npmjs.org/phantomjs-prebuilt 133ms (from cache)
46 silly pacote range manifest for validator@^5.0.0 fetched in 133ms
47 silly resolveWithNewModule validator@5.7.0 checking installable status
48 silly pacote range manifest for underscore@^1.8.2 fetched in 134ms
49 silly resolveWithNewModule underscore@1.8.3 checking installable status
50 silly pacote range manifest for phantomjs-prebuilt@^2.1.12 fetched in 135ms
51 silly resolveWithNewModule phantomjs-prebuilt@2.1.16 checking installable status
52 silly saveTree FoodLionDotCom@0.1.0
52 silly saveTree ├─┬ grunt-browserify@3.8.0
52 silly saveTree │ ├── async@0.9.2
52 silly saveTree │ ├─┬ browserify@10.2.6 ( so many lines like this) 
53 verbose stack Error: exited with error code: 128
53 verbose stack     at ChildProcess.onexit (/usr/local/lib/node_modules/npm/node_modules/mississippi/node_modules/end-of-stream/index.js:39:36)
53 verbose stack     at emitTwo (events.js:126:13)
53 verbose stack     at ChildProcess.emit (events.js:214:7)
53 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
54 verbose cwd /Users/1557664/Documents/FoodLion/foodlionIJ
55 verbose Darwin 16.7.0
56 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "grunt-wcag-accessibility"
57 verbose node v8.9.4
58 verbose npm  v5.6.0
59 error Error while executing:
59 error /usr/local/bin/git ls-remote -h -t ssh://git@github.com/squizlabs/HTML_CodeSniffer.git
59 error
59 error fatal: unable to access '/var/root/.gitcinclude': Permission denied
59 error
59 error exited with error code: 128

Do i need any permission to access this project?

**Comment by bpearson** [comment]
It appears to be a file permission problem with this message: npm ERR! fatal: unable to access '/var/root/.gitcinclude': Permission denied

You either need to run as root (aka sudo npm install grunt-wcag-accessibility) or change the perms on /var/root to include the current user (in which case you need root access or someone with root access).
